internal/bizdatatypes/core.go

```diff
@@ -35,14 +35,14 @@ var URLValidator = db.Code{
 `,
 }
 
-var EmailAddress = db.Datatype{
+var EmailAddress = db.Composit{
```

Comment:
The name Composit doesn't really say what it is. maybe just datatypestorage or something like that?